force-app/main/default/classes/DMLOperations.cls

```diff
@@ -160,14 +186,24 @@ public with sharing class DMLOperations {
 	 */
 	public static Account upsertAccount(String accountName) {
 		// Query for the Account
-		List<Account> accounts = [SELECT Id FROM Account WHERE Name = :accountName];
+		List<Account> accountsList = [SELECT Id FROM Account WHERE Name = :accountName];
 
-		// if the Account exists, update the description
+        Account upsertedAccount = new Account();
+
+        if(accountsList.isEmpty() == true){
```

Comment:
Here we don't need to assess if it is true or false. Instead, the expression itself (`accountsList.isEmpty()`) will return true or false:
```suggestion
        if(accountsList.isEmpty()){
```
And if wanted to say NOT empty, we could write it like this:
```suggestion
        if(!accountsList.isEmpty()){
```
Let me know if you have any questions on this.